user_guide_src/source/database/query_builder.rst

```diff
@@ -167,6 +167,8 @@ the resulting field.
 .. literalinclude:: query_builder/014.php
    :lines: 2-
 
+.. _query-builder-select-subquery:
```

Comment:
@iRedds The label must  precede a section header.

`**$builder->selectSubquery()**` is not a section.